populate.js

```diff
@@ -16,14 +16,15 @@ const populateDb = () => {
 const savePhotos = async (primaryUrls, productPhotosUrls) => {
   let dbRecords = [];
   let featuresPhotoSizes = [[960, 832], [960, 400], [960, 123], [960, 832], [700, 568], [700, 568], [700, 568], [960, 832], [547, 454], [300, 270], [300, 270], [300, 270], [50, 50], [50, 50], [50, 50], [50, 50], [50, 50]];
```

Comment:
I would change this line of code to be multiple lines instead of one long line so that everything can be seen without having to scroll.